torso-bundle.js

```diff
@@ -3523,10 +3727,10 @@
         injectionSite = this.$el;
       args = args || {};
       this._modelName = args.childModel || 'model';
-      this._collection = args.collection;
-      this._childView = args.childView;
-      this._template = args.template;
-      this._childrenContainer = args.childrenContainer;
+      this.collection = args.collection || this.collection;
```

Comment:
This lines the .collection property of backbone up with the collection associated with the ListView.